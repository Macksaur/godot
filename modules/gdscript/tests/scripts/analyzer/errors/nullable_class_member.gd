var foo: String? = null


func test():
    print(foo.begins_with("")) # Should error