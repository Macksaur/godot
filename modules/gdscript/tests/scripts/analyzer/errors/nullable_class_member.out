GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable "foo" to call "begins_with()".
